### PR TITLE
Updating Getting started to use the single Image

### PR DESCRIPTION
**Pull request by kumare3**

### CODE REVIEW

**Review by katrogan — DISMISSED** [review]

File: rsts/howto/sandbox.rst [hunk 1]
```diff
@@ -21,8 +21,95 @@ Flyte can be run using a Kubernetes cluster only. This installs all the dependen
...
-Refer to :ref:`tutorials-getting-started-run-on-flyte`.
+Ensure ``kubectl`` is installed. Follow `kubectl installation docs <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_. On Mac::
+
+    brew install kubectl
+
+
+
+.. tabs::
+
+    .. tab:: Use flyte-sandbox docker image (Recommended)
```
Comment (EngHabu) [inline rsts/howto/sandbox.rst:32]: Maybe just these as titles to make them fit in "regular" screen sizes?
"Docker, K3d, Docker Desktop, Minikube, EKS/GKE"

File: rsts/howto/sandbox.rst [hunk 2]
```diff
@@ -21,8 +21,95 @@ Flyte can be run using a Kubernetes cluster only. This installs all the dependen
...
+
+    .. tab:: Install Flyte in EKS/GKE etc
+
+        Refer to the :ref:`howto-sandbox-dedicated-k8s-cluster` guide.
+
+        Once you've deployed flyte sandbox to a cloud provider, read on below to run your first workflow remotely.
+
 
 
 .. _howto-sandbox-dedicated-k8s-cluster:
```
Comment (EngHabu) [inline rsts/howto/sandbox.rst:115]: Do we still need this section here?

File: rsts/howto/sandbox.rst (hunk 1 above)
Comment (EngHabu) [reply rsts/howto/sandbox.rst:32]: Given there is a separate section below for EKS/GKE, maybe we don't need the tab here at all?

**Review by EngHabu — COMMENTED** [review]
In the sandbox doc, this line has a broken ref: https://github.com/flyteorg/flyte/pull/775/files#diff-48b6cfff8f3fe18b27ba0354b545a8da0db7c0e9d2a03d2d6fb714fb8b7245a3R136

File: rsts/howto/sandbox.rst (hunk 2 above)
Comment (kumare3) [reply rsts/howto/sandbox.rst:115]: I think there will be folks that will invariably try this out

**Review by EngHabu — APPROVED** [review]

**Pull request merged**